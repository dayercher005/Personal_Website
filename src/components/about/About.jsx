import AboutCard from "./AboutCard.jsx"

export default function About() {

    return(
        <div id="About" className="flex justify-center py-150">
            <AboutCard></AboutCard>
            
        </div>
    )
}