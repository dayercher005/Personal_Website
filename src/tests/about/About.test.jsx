import { describe, it, expect } from "vitest";
import { render} from "@testing-library/react";
import About from '/src/components/about/About.jsx'

describe("About component", () => {

    it("renders About Section", () => {
        const { container } = render(<About/>)
        expect(container).toMatchSnapshot()
    })
})