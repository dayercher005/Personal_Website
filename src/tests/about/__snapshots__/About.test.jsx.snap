// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About component > renders About Section 1`] = `
<div>
  <div
    class="flex justify-center py-60"
    id="About"
  >
    <div>
      <h2
        class="inline-block text-4xl font-bold my-2 border-b-blue-400 border-b-5"
      >
        About me
      </h2>
      <p
        class="font-medium text-lg max-w-3xl text-gray-500"
        data-testid="aboutParagraph"
      >
        Currently pursuing a Computer Science Degree in Singapore. As a Computer Science student, I see learning as a continuous journey to evolve myself personally and professionally. Beyond the classroom, I\`m open to exploring opportunities to grow as a developer through hands-on projects and internships, where I can learn from real-world challenges and contribute to impactful software. 
      </p>
    </div>
  </div>
</div>
`;
